packages/wrangler/src/__tests__/helpers/msw/handlers/deployments.ts

```diff
@@ -53,17 +53,68 @@ export const mswSuccessDeployments = [
 	),
 ];
 
-export const mswSuccessLastDeployment = [
+export const mswSuccessDeploymentScriptMetadata = [
 	rest.get(
 		"*/accounts/:accountId/workers/services/:scriptName",
 		(_, res, ctx) => {
 			return res.once(
 				ctx.json(
 					createFetchResult({
-						default_environment: { script: { last_deployed_from: "wrangler" } },
+						default_environment: {
+							script: { last_deployed_from: "wrangler", tag: "MOCK-TAG" },
```

Comment:
Does EWC always provide a `tag` property in the result?

Reply:
As far as I have seen, in examples I have gotten in tickets. I can double check, with the API contract.

Reply:
Yes

```json
{
  "result": {
    "id": "testss",
    "default_environment": {
      "environment": "wees",
      "created_on": "2022-11-18T16:40:48.50545Z",
      "modified_on": "2023-01-20T18:08:47.464024Z",
      "script": {
        "created_on": "2022-11-18T16:40:48.50545Z",
        "modified_on": "2023-01-20T18:08:47.464024Z",
        "id": "testss",
        "tag": "4918777f032148a59feb6d2c2b770e34",
        "tags": null,
        "deployment_id": "",
        "logpush": false,
        "etag": "13a3240e8fb414561b0366813b0b8f42b3e6cfa0d9e70e99835dae83d0d8a794",
        "handlers": [
          "fetch"
        ],
        "last_deployed_from": "api",
        "usage_model": "bundled"
      }
    },
    "created_on": "2022-11-18T16:40:48.50545Z",
    "modified_on": "2023-01-20T18:08:47.464024Z",
    "usage_model": "",
    "environments": [
      {
        "environment": "wees",
        "created_on": "2022-11-18T16:40:48.50545Z",
        "modified_on": "2023-01-20T18:08:47.464024Z"
      }
    ]
  },
  "success": true,
  "errors": [],
  "messages": []
}
```